### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AVyha**
image
image
image
image
image
image
image

### CODE REVIEW

File: taxi/models.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=50)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE
+    )
+    drivers = models.ManyToManyField(
+        "Driver"
```
Comment (HStakhiv) [inline taxi/models.py:20]: I think there should be an `AUTH_USER_MODEL` instead of `Driver`

**Review by HStakhiv — CHANGES_REQUESTED** [review]
Use `black` to fix single quotes.
Why did not download the migration?

**Comment by AVyha** [comment]
> Why did not download the migration?
Im forget about migrations :)

**Review by HStakhiv — APPROVED** [review]
Looks good

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+*.db
+*.sqlite3
+*.pyc
+.idea
```
Comment (Y-Havryliv) [inline .gitignore:4]: add blank line to the end of the file

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**